stratum/hal/bin/barefoot/start-stratum.sh

```
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# Copyright 2020-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+KDRV_PATH=${KDRV_PATH:-/usr/lib/modules/bf_kdrv.ko}
+PLATFORM=${PLATFORM:-x86-64-accton-wedge100bf-32x-r0}
+
+if [ -f "/etc/onl/platform" ]; then
+    PLATFORM="$(cat /etc/onl/platform)"
+fi
+
+# Setup port map
+rm /usr/share/port_map.json || true
```

Use `rm -f` to ignore error?